src/databricks/labs/ucx/source_code/linters/table_creation.py

```diff
@@ -57,13 +57,13 @@ def get_advice_span(self, node: NodeNG) -> Range | None:
         # Check 4: check presence of the format specifier:
         #   Option A: format specifier may be given as a direct parameter to the table-creating call
         #   >>> df.saveToTable("c.db.table", format="csv")
-        format_arg = Tree.get_arg(node, self.format_arg_index, self.format_arg_name)
-        if format_arg is not None and not Tree.is_none(format_arg):
+        format_arg = TreeHelper.get_arg(node, self.format_arg_index, self.format_arg_name)
+        if format_arg is not None and not TreeHelper.is_none(format_arg):
             # i.e., found an explicit "format" argument, and its value is not None.
             return None
         #   Option B. format specifier may be a separate ".format(...)" call in this callchain
         #   >>> df.format("csv").saveToTable("c.db.table")
-        format_call = Tree.extract_call_by_name(node, "format")
+        format_call = TreeHelper.extract_call_by_name(node, "format")
```

Comment:
given the number of renames, the `TreeHelper` introduction is unnecessary

Reply:
This is to avoid linting error with public methods > 20